README.md

```diff
@@ -36,16 +30,4 @@ Project resources
 - [Gitter.im Chat Room](https://gitter.im/Netflix/security_monkey)
 - [CloudAux](https://github.com/Netflix-Skunkworks/cloudaux)
 - [PolicyUniverse](https://github.com/Netflix-Skunkworks/policyuniverse)
-- [Troubleshooting](docs/troubleshooting.md)
-
-
-Instance Diagram
----------------
-The components that make up Security Monkey are as follows (not AWS specific):
-![diagram](docs/images/sm_instance_diagram.png)
-
-
-Access Diagram
```

Comment:
This as well.

Reply:
Done: https://github.com/Netflix/security_monkey/pull/950/commits/51c9079f5b3080a0f78bf978725074d0dd47e38c